src/components/utils/dynamicLoading.ts

```diff
@@ -0,0 +1,12 @@
+type OnloadCallback = () => void
+
+export const loadExternalScript = (src: string, onload: OnloadCallback) => {
```

Comment:
We're already using [loading lazy components](https://github.com/onfido/onfido-sdk-ui/blob/development/src/components/ActiveVideo/Lazy.tsx) in the Web SDK using dynamic imports. Have you considered this option rather than using the good old script injection?

Reply:
Thanks, I was fully aware of our lazy loading stuff. The reason for not going this way was it's not about lazy loading. It's about entirely control of the release cycles. As a separate developed module, we don't fancy being stuck with the Web SDK releases. Furthermore, there's already a plan to offer this module outside of the SDK world, so we need to keep it like this for the sake of flexibility.

Reply:
Ok, yes, that makes sense considering the RFC. Thanks!